### ISSUE TITLE
MERRA2 climatology updated from 2003-2014 mean to 2013-2024 mean

### ISSUE THREAD

**Issue by AnningCheng-NOAA** [NOAA-EMC/global-workflow#2887]
### Target fix directory

aer

### Type of change

- [ ] Update existing files
- [ ] Remove files
- [X] Add new files to existing directory
- [ ] Add new fix directory

### Any workflow changes needed?

copy and link 2013-2024 mean data set  instead of 2003-2014 mean 

### Related issues

https://github.com/ufs-community/ufs-weather-model/issues/2272
https://github.com/ufs-community/ufs-weather-model/pull/2273

### Pickup location

/scratch2/NAGAPE/epic/UFS-WM_RT/NEMSfv3gfs/input-data-20240501/FV3_input_data_INCCN_aeroclim/MERRA2_y14_24

### Size

8854

**Comment by WalterKolczynski-NOAA** [comment]
Copied to fix on hera as `merra2.aerclim.2014-2023.mMM.nc` to match the existing naming convention. Please confirm and then I will sync to other machines.

**Comment by WalterKolczynski-NOAA** [comment]
```
[role.glopara@hfe05 20220805]$ pwd
/scratch1/NCEPDEV/global/glopara/fix/aer/20220805
[role.glopara@hfe05 20220805]$ ls
merra2.aerclim.2003-2014.m01.nc  merra2.aerclim.2003-2014.m10.nc  merra2.aerclim.2014-2023.m07.nc   merra2C.aerclim.2003-2014.m04.nc
merra2.aerclim.2003-2014.m02.nc  merra2.aerclim.2003-2014.m11.nc  merra2.aerclim.2014-2023.m08.nc   merra2C.aerclim.2003-2014.m05.nc
merra2.aerclim.2003-2014.m03.nc  merra2.aerclim.2003-2014.m12.nc  merra2.aerclim.2014-2023.m09.nc   merra2C.aerclim.2003-2014.m06.nc
merra2.aerclim.2003-2014.m04.nc  merra2.aerclim.2014-2023.m01.nc  merra2.aerclim.2014-2023.m10.nc   merra2C.aerclim.2003-2014.m07.nc
merra2.aerclim.2003-2014.m05.nc  merra2.aerclim.2014-2023.m02.nc  merra2.aerclim.2014-2023.m11.nc   merra2C.aerclim.2003-2014.m08.nc
merra2.aerclim.2003-2014.m06.nc  merra2.aerclim.2014-2023.m03.nc  merra2.aerclim.2014-2023.m12.nc   merra2C.aerclim.2003-2014.m09.nc
merra2.aerclim.2003-2014.m07.nc  merra2.aerclim.2014-2023.m04.nc  merra2C.aerclim.2003-2014.m01.nc  merra2C.aerclim.2003-2014.m10.nc
merra2.aerclim.2003-2014.m08.nc  merra2.aerclim.2014-2023.m05.nc  merra2C.aerclim.2003-2014.m02.nc  merra2C.aerclim.2003-2014.m11.nc
merra2.aerclim.2003-2014.m09.nc  merra2.aerclim.2014-2023.m06.nc  merra2C.aerclim.2003-2014.m03.nc  merra2C.aerclim.2003-2014.m12.nc
```
